build_debian.sh

```diff
@@ -43,6 +43,15 @@ HOSTNAME=sonic
 DEFAULT_USERINFO="Default admin user,,,"
 BUILD_TOOL_PATH=src/sonic-build-hooks/buildinfo
 TRUSTED_GPG_DIR=$BUILD_TOOL_PATH/trusted.gpg.d
+## Remapped docker usernames
+REMAPREDIS=remapredis
+REMAPROOT=remaproot
+## Start and Range ID for the default user defined in /etc/subuid and /etc/subgid
+START_ID=100000
+RANGE_ID=65536
```

Comment:
Do you mean these magic numbers are from /etc/subuid and /etc/subgid, not you randomly pick?
If yes, could you get the number from files programmatically?

Reply:
Not randomly pick, changed to get number from files programmatically, thank you!